cms/check_rules_states_by_account

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+"""
+Probe that counts the number of stuck, replicating rules and waiting for approval rules, clasiffied by account.
+The accounts used to classify the rules are sotred in the accounts list.
+"""
+
+from rucio.common.types import InternalAccount
+from rucio.db.sqla.constants import RuleState
+import sys
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+from rucio.common.config import config_get
+from rucio.core import monitor
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+from rucio.db.sqla.util import get_count
+
+transfer_ops = InternalAccount('transfer_ops')
+wmcore_output = InternalAccount('wmcore_output')
+wma_prod = InternalAccount('wma_prod')
+wmcore_transferor = InternalAccount('wmcore_transferor')
+crab_tape_recall = InternalAccount('crab_tape_recall')
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+
+PROM_SERVERS = config_get('monitor', 'prometheus_servers', raise_exception=False, default='')
```

Comment:
Likewise I don't see that prometheus is actually used. You should use it if you can.